include/GafferSceneUI/ContextAlgo.h

```diff
@@ -102,6 +104,21 @@ GAFFERSCENEUI_API IECore::PathMatcher expandDescendants( Gaffer::Context *contex
 /// Clears the currently expanded paths
 GAFFERSCENEUI_API void clearExpansion( Gaffer::Context *context );
 
+/// VisibleSet
+/// ==========
+
+/// On top of the expansion mechanics described above (`expansions`), a VisibleSet
+/// allows subsets of the scene to be defined as _always_ rendered (`inclusions`),
+/// or _never_ rendered (`exclusions`).
```

Comment:
I think we should document the VisibleSet as being the one true source of information, rather than being something "on top of" the expanded paths. I'd be in favour of deprecating the expanded paths stuff entirely, and just documenting it as being provided for temporary backwards compatibility. There is some useful documentation of the expansion semantics in amongst it, but that could be moved to the VisibleSet documentation.

Reply:
I've revised the existing expansion documentation to be more VisibleSet centric in b6fe7b7, let me know if that reads better to you. I've added a comment to the expanded paths stuff mentioning its now legacy existence, but haven't gone so far as to start marking things as deprecated.